src/main/java/controller/BookController.java

```diff
@@ -16,40 +22,47 @@
 import org.springframework.web.bind.annotation.RestController;
 import service.BookService;
 
+@Tag(name = "Book store API", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
+@Validated
 @RequestMapping("/books")
 public class BookController {
     private final BookService bookService;
 
     @ResponseStatus(HttpStatus.OK)
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books", description = "Get list of books")
+    public List<BookDto> getAll(Pageable pageable) {
```

Comment:
consider using `@PageableDefault` in case client did not specify pagination params

also you may annotate it with `@ParameterObject` so that it is nicely represented on Swagger